index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <main id="main">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+<h1 id="title">"Kaneki Kun"</h1>
+  <body>
+    "Hello World, we are software developers!"
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div id="img-div">
+      <img id="image" src="https://cdn.pixabay.com/photo/2018/10/14/00/15/ken-3745445_960_720.jpg" alt="Image of kaneki">
+      <h3 id="img-caption">"When kaneki was child"</h3>
+      <p id="tribute-info">Kaneki kun was a man and after an accident he turn into ghoul
+        <ol id="facts"> Facts about kaneki
+          <li>He was kind and innocent</li>
+          <li>He did not loose his humanity</li>
+          <li>He became strong to protect his friends</li>
+          <li>He never killed humans</li>
+          <li>He gave everything for the sake of his loved</li>
+        </ol>
+      </p>
+      <a id="tribute-link" href="https://tokyoghoul.fandom.com/wiki/Ken_Kaneki" target="_blank">Kaneki's bio</a>
+    </div>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Please, do not use unused code as comments in your codebase. Unused code as comments makes your code not clean☺️.